src/main/java/uk/gov/hmcts/reform/sscs/thirdparty/documentmanagement/DocumentManagementService.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.sscs.thirdparty.documentmanagement;
+
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.multipart.MultipartFile;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.document.DocumentUploadClientApi;
+import uk.gov.hmcts.reform.document.domain.UploadResponse;
+
+@Slf4j
+@Service
+public class DocumentManagementService {
```

Comment:
Pretty sure we have the same thing in common library so might want to use that if it's straight forward?